index.js

```diff
@@ -0,0 +1,66 @@
+class TinyPagination {
+
+    pageNo = 1
+
+    constructor(pageSize, data) {
+        this.pageSize = pageSize
```

Comment:
Hello :)

I think I would add, in the variable pageSize, the type of file size. For example GB, MB, maybe something else that you are referring to. So something like pageSizeInMB, or anything you think that would make it precise.

Reply:
Thanks for your comments!
The `pageSize` is the number of data that the user wants to display on each page. Maybe I name it in the wrong way.